ackhandler/sent_packet_handler.go

```diff
@@ -333,7 +333,11 @@ func (h *sentPacketHandler) SendingAllowed() bool {
 			h.bytesInFlight,
 			h.congestion.GetCongestionWindow())
 	}
-	return !(congestionLimited || maxTrackedLimited)
+	// Workaround for #555:
+	// Always allow sending of retransmissions. This should probably be limited
+	// to RTOs, but we currently don't have a nice way of distinguishing them.
```

Comment:
We can allow retransmission of (1) the oldest two outstanding packets (per RTO), (2) of all outstanding packets, (3) of the oldest two outstanding packets due to RTO (per RTO), or (4) of all outstanding packets due to RTO. This CL implements (2), but I don't think it's the right solution.

If you don't like merging this I'm fine with waiting, but I thought it'd be nice to have a partial fix out asap, as people are waiting for it (#555).